CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ possible.
 ## Getting started
 
 Most contributions will involve working with the neutrino-dev codebase. Please refer to the [development
-documentation](/contributing/development.md) for technical details on getting started.
+documentation](/docs/contributing/development.md) for technical details on getting started.
```

Comment:
Links for gitbook have to be relative in order for them to work on the docs site as well as github. Change to: `./development.md`.